### PR TITLE
Add privacy policy for packages that talk to Google APIs

### PR DESCRIPTION
**Pull request by jennybc**
Needed so we can verify the gargle project.

TODOs:

  * [x] Settle on a project / "app" name. Current placeholder in doc is GARGLE_PROJECT. Current OAuth consent screen shows "tidyverse api packages". *I went with "Tidyverse API Packages".*
  * [x] Settle on an email. Currently it goes to "XYZ@rstudio.com" but that should probably change, probably to match the project or "app" name we choose above. *Decision made; alias created.*
  * [ ] ~Determine where to put within the tidyverse.org site and how we are going surface it. Presumably most traffic will come from clicking on a direct link in an OAuth consent screen. But I imagine it has to be *possible* to navigate to it directly.~
  * [ ] Once this is merged and has a stable URL, incorporate that link visibly in each package's pkgdown site, if such exists, or README otherwise (gmailr). *Can't happen in this PR, obviously.*
  * [x] Decide if we want the tidyverse logo on our consent screen. *I don't think we have a choice. In any case, I can't figure out how to remove the logo, i.e. without replacing it with another image.*

Background:

  * [OAuth API Verification FAQ](https://support.google.com/cloud/answer/9110914)

Key info:

> The Privacy Policy must be visible to users, hosted within the domain of your website, and linked to the OAuth consent screen on the Google API Console.
> The Privacy Policy must be compliant with the [Google API User Data Policy](https://developers.google.com/terms/api-services-user-data-policy) and the [Limited Use requirements](https://developers.google.com/terms/api-services-user-data-policy#additional-requirements-for-specific-api-scopes). It must disclose the manner in which your application accesses, uses, stores, or shares Google user data. Your use of Google user data must be limited to the practices disclosed in your published Privacy Policy.

### CODE REVIEW

**Comment by jennybc** [comment]
FYI @jcheng5

**Comment by jennybc** [comment]
Link to rendered preview: https://deploy-preview-315--tidyverse-org.netlify.com/google_privacy_policy/

File: content/google_privacy_policy.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+At no time does the GARGLE_PROJECT receive your data or the permission to access your data. The GARGLE_PROJECT can only see information about aggregate usage of tokens associated with its OAuth client, such as which APIs and endpoints are being used. If you provide a service account token or obtain a token with your own OAuth client, your usage will not appear in the usage data that Google attributes to the GARGLE_PROJECT.
+
+### Storing user data
+
+By default, your token is stored to a local file, such as `~/.R/gargle/gargle-oauth`. See the documentation for [`gargle::gargle_options()`](https://gargle.r-lib.org/reference/gargle_options.html) and [`gargle::credentials_user_oauth2()`](https://gargle.r-lib.org/reference/credentials_user_oauth2.html) for information on how to control the location of the token cache or suppress token caching, globally or at the individual token level. All user data is stored on your local machine. **Use caution when using these packages on a shared machine**.
```
Comment (hadley) [inline content/google_privacy_policy.md]: Here, I would first reinforce that we never send data elsewhere, but that access credentials are stored locally.

File: content/google_privacy_policy.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<!--
+Heavily inspired by:
+https://pandas-gbq.readthedocs.io/en/latest/privacy.html
+https://pydata-google-auth.readthedocs.io/en/latest/privacy.html
+-->
+
+The tidyverse team maintains several packages that make it easier to work with Google APIs from R:
+
+  * gargle: <https://gargle.r-lib.org>
+    - Infrastructure package with general helpers for Google APIs, including auth
```
[the remark this answers is not in the document]
Comment (jennybc) [reply content/google_privacy_policy.md]: There is verbiage in the instructions that makes it seem like we should be very clear why we this project is associated with so many restricted scopes. We are literally 15 for 15 as far as restricted scopes go (basically Gmail and Drive).

> How can I make sure the verification process is as streamlined as possible?
> 
> Make sure that each scope that you're requesting has an explanation for its use/need for the project, as well as a justification for why a narrower scope would be insufficient. 

Therefore, I was trying to make it clear that our packages meet the requirements for "Application Type" here:

https://developers.google.com/terms/api-services-user-data-policy#additional-requirements-for-specific-api-scopes

[the remark this answers is not in the document]
Comment (hadley) [reply content/google_privacy_policy.md]: In that case, I would move those further down the page, and organise by scope.

[the remark this answers is not in the document]
Comment (jennybc) [reply content/google_privacy_policy.md]: > I think you should just link to the packages here, with describing what they do.

Oh wait, are you saying to drop gargle from this list?
Or to not hyperlink to the APIs?

**Comment by jennybc** [comment]
I think this is ready (other than the TODOs in the description), which I can make executive decisions about, if no one has more input.

**Pull request merged**